ccloud/beginner-cloud/ccloud_stack_spin_up.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#########################################
+# This script uses real Confluent Cloud resources.
+# To avoid unexpected charges, carefully evaluate the cost of resources before launching the script and ensure all resources are destroyed after you are done running it.
+#########################################
+
+
+# Source library
+. ../../utils/helper.sh
+
+check_ccloud_version 1.0.0 || exit 1
+check_jq || exit 1
+check_ccloud_logged_in || exit 1
+
+prompt_continue_cloud_demo || exit 1
+
+enable_ksql=false
+read -p "Do you also want to create a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
+echo
+if [[ $REPLY =~ ^[Yy]$ ]]
+then
+  enable_ksql=true
+fi
+
+echo
+echo "Spin up..."
+cloud_create_demo_stack $enable_ksql
+
+echo
+echo "Validating..."
+SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
+CONFIG_FILE=/tmp/client-$SERVICE_ACCOUNT_ID.config
```

Comment:
IMO we should stop writing outside the demo repositories to system level directories.  We could write these to a local directory, where we know the user already has write access, and then add that to .gitignore.  For instance, maybe a good folder would be ./stack-configs and we could add ./stack-configs/* to a local .gitignore.

Reply:
@rspurgeon 💯 agree and fixed in the latest commit.  Please let me know if further polishing is required.